pom.xml

```diff
@@ -281,7 +281,7 @@
         <configuration>
           <!-- Ensure junit tests can access recently compiled binaries -->
           <systemPropertyVariables>
-            <jssc.boot.library.path>${project.build.directory}/cmake/natives/</jssc.boot.library.path>
+            <jssc.boot.library.path>${project.build.directory}/target/cmake/natives/</jssc.boot.library.path>
```

Comment:
It's actually wrong now
```WARNING jssc.DefaultJniExtractorStub: Boot path /Users/runner/work/jssc/jssc/target/target/cmake/natives/ not found, falling back to default extraction behavior.```

Reply:
> It's actually wrong now
> 
> ```
> WARNING jssc.DefaultJniExtractorStub: Boot path /Users/runner/work/jssc/jssc/target/target/cmake/natives/ not found, falling back to default extraction behavior.
> ```

Yeah it won't work in the `pom.xml`, we it needs the suffix too, e.g. `windows_64`, etc.

Reply:
Reverted back to setting it in the test.